inc/computer_softwarelicense.class.php

```diff
@@ -542,7 +545,7 @@ static function showForLicense(SoftwareLicense $license) {
             if ($key[0] == '_') {
                $header_end .= "<th>$val</th>";
             } else {
-               $header_end .= "<th".($sort == "`$key`" ? " class='order_$order'" : '').">".
+               $header_end .= "<th>".
```

Comment:
I think we lose something here:

![image](https://user-images.githubusercontent.com/418844/57600639-a8bd5e80-755a-11e9-9df8-e7002a6bba45.png)